L1Trigger/TrackFindingTMTT/src/KFbase.cc

```diff
@@ -812,8 +814,8 @@ namespace tmtt {
     }
 
     bool ambiguous = ambiguityMap[kfEtaReg][kfLayer];
-
-    //bool ambiguous = false;
+    if (!Settings_->KFUseMaybeLayers()) ambiguous = false;
```

Comment:
More CPU efficient (and simpler):
bool ambiguous = false;
if (Settings_->KFUseMaybeLayers()) ambiguous = ambiguityMap[kfEtaReg][kfLayer];